aldryn_search/helpers.py

```diff
@@ -35,12 +36,14 @@ def get_cleaned_bits(data):
 
 
 def get_plugin_index_data(base_plugin, request):
+    excluded_plugins = getattr(settings, 'ALDRYN_SEARCH_PLUGINS_EXCLUDE', [])
     text_bits = []
 
     instance, plugin_type = base_plugin.get_plugin_instance()
 
-    if instance is None:
-        # this is an empty plugin
+    if instance is None or hasattr(instance, '_meta') and hasattr(instance._meta,
```

Comment:
change to `if instance is None or instance.plugin_type in EXCLUDED_PLUGINS`

Reply:
I did it first time, but some more tests became failed with object "Options" or something like this

Reply:
Ok, i'll change it